src/app/integrations/list/list.component.scss

```diff
@@ -85,9 +102,130 @@
             }
           }
 
-          .label {
-            padding: 6px;
+          // Draft State
+
+          .draft {
+            display: inline-block;
+            padding: 0 25px;
+
+            .label {
+              cursor: pointer;
+              padding: 6px;
+            }
+          }
+
+
+          // Other States - Toggle Button
+
+          .switch_wrap input[type="checkbox"] {
```

Comment:
We shouldn't be using these either, but this was literally a copy and paste from an example. :/ *runs and hides*